public/javascripts/map.js

```diff
@@ -462,7 +501,6 @@ define(['settings', 'jquery', 'brush'], function (settings, jquery, brush) {
 
                 cursorContext.fill();
                 cursorContext.stroke();
-                requestAnimationFrame(setupCursorTracking)
```

Comment:
@apclary Is there any reason why the event handlers got set after each cursor draw? Resulted in huge performance problems for me.

Reply:
Probably just poor coding on my part, I was pretty new to JavaScript when I wrote this.